File: android/using-lazy-in-kotlin-to-bind-android-views/app/src/main/java/com/lordcodes/lazyviewbindingarticle/findviewbyid/PlanningActivity.java
package com.lordcodes.lazyviewbindingarticle.findviewbyid;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

import com.lordcodes.lazyviewbindingarticle.R;

import androidx.appcompat.app.AppCompatActivity;

public class PlanningActivity extends AppCompatActivity {
    private TextView planningText;
    private ImageView appIcon;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_planning);
        // Before: Needed to cast
        planningText = (TextView) findViewById(R.id.planning_text);
        // Now: No longer need to
        appIcon = findViewById(R.id.app_icon);

        planningText.setText("Hello!");
    }
}

File: android/using-lazy-in-kotlin-to-bind-android-views/app/src/main/kotlin/com/lordcodes/lazyviewbindingarticle/kotlinandroidextensions/PlanningActivity.kt
package com.lordcodes.lazyviewbindingarticle.kotlinandroidextensions

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.lordcodes.lazyviewbindingarticle.R

import kotlinx.android.synthetic.main.activity_planning.*

class PlanningActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_planning)

        planning_text.text = "Hello"
    }
}